appmap/_implementation/configuration.py

```diff
@@ -52,7 +69,11 @@ def in_set(name, which):
 
 
 def function_in_set(fn, which):
-    class_name, fn_name = utils.split_function_name(fn)
+    class_name, fn_name = split_function_name(fn)
+    if class_name is None:
+        # Is this really the right thing to do?
```

Comment:
@apotterri what is the aim here?

Reply:
I'm not sure how bare functions (i.e. functions that aren't in a class) are supposed to be represented in an AppMap. Until I figure it out, this just skips instrumenting them.

Reply:
I've updated the comment to indicate what's going on here.